commands/addrole.js

```diff
@@ -0,0 +1,127 @@
+const Discord = require("discord.js")
+const reaction_message_model = require("../models/reaction_roles_message.js")
+const mongoose = require('mongoose');
+const conf = require('../conf.js');
+mongoose.connect(conf.mongodbs, { useNewUrlParser: true })
+
+/**
+ * Adds a new role to any message in the server
+ */
+module.exports = async (bot, message, args) => {
+    // check for help
+    if (!args[2] || args[0] == "help") {
+        let help = new Discord.RichEmbed()
+            .setColor("#FFFF00")
+            .addField("Usage", "addemo <message_id> <emote> <role>")
+            .addField("More info", "Adds a new role to any message in the server.")
+        return message.channel.send(help).then(r => r.delete(30000))
+    }
+    // check if command issuer has perms
+    if(!message.member.hasPermission("MANAGE_ROLES")){
+        message.reply("You don't have permission").then(async m=>{
```

Comment:
Unless necessary (i.e. to avoid rebinding of this) arrow functions are discouraged for anything more than a quick one-line filter or transform. Avoid arrow functions for anything more than a one-liner.
```suggestion
        message.reply("You don't have permission").then(async function(m) {
```